base/remote/R/remote.copy.to.R

```diff
@@ -22,6 +22,9 @@ remote.copy.to <- function(host, src, dst, options = NULL, delete = FALSE, stder
   if (as.logical(delete)) {
     args <- c(args, "--delete")
   }
+  if (!is.character(host) | length(host) > 1) {
+    PEcAn.logger::logger.severe("`host` should be a length 1 character vector, not ", class(host), " length ", length(host))
```

Comment:
The rest of the function is expecting `host` to have at least 3 entries -- see the @examples block above.

Reply:
ah right, I forgot that `settings$host` has a bunch of stuff in it.

Reply:
I think I'd advise testing that each of the needed items _inside_ `host` are non-NULL and have the needed format... then you'll correctly fail on cases like `host = list(naem = ...)` as well as on `host = NULL`.